internal/core/model/adapters.rs

```diff
@@ -341,11 +349,19 @@ where
         Self(container)
     }
 
+    /// Manually reapply the filter. You need to run this e.g. if the filtering function uses
+    /// mutable state and it has changed.
```

Comment:
maybe the doc should say that this is deprecated, with a link to the new version